src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,23 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Random random = new Random();
+
+    @Override
+    public List<? extends Machine> get() {
+        Bulldozer bulldozer1 = new Bulldozer(random.nextInt());
+        Bulldozer bulldozer2 = new Bulldozer(random.nextInt());
+        Bulldozer bulldozer3 = new Bulldozer(random.nextInt());
+        List<Bulldozer> bulldozers = new ArrayList<>();
```

Comment:
probably it's better to move list of machines as a class variable, to not create it in get method

Reply:
That's a bad idea, cause the class should contain fields that describe it. For example, can bulldozer contain a list of bulldozers inside in real life? Probably not.